app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
Code Efficiency issue: It's recommended to use the class name to get access to the class attributes. Use `Animal.alive` instead of `self.alive`.